v1/swagger.yaml

```diff
@@ -161,6 +161,7 @@ paths:
           name: subscriptionID
           schema:
             type: string
+            format: uuid
           required: true
           description: The ID of the subscription to receive
```

Comment:
Maybe change the wording to :
The universal unique ID of the subscription to receive
?

Reply:
done